app/javascript/packs/shared/navigation/index.jsx

```diff
@@ -43,23 +44,36 @@ class Navigation extends Component {
     this.state = {
       showActivitesModal: false,
       page: "",
-      currentTeam: { id: 0 }
+      current_team: { id: 0 }
```

Comment:
I don't know how I feel about violating naming conventions and using snake case throughout some js files. But hey it reads much better :)